rpc/lib/server/handlers.go

```diff
@@ -538,7 +538,7 @@ func (wsc *wsConnection) writeRoutine() {
 				return
 			}
 		case msg := <-wsc.writeChan:
-			jsonBytes, err := json.Marshal(msg)
+			jsonBytes, err := json.MarshalIndent(msg, "", "  ")
```

Comment:
Shouldn't it be the responsibility of a client if it wants JSON to look pretty? `curl ... | json_pp` for example